server/container_create.go

```diff
@@ -548,13 +547,6 @@ func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerReq
 		return nil, fmt.Errorf("CreateContainer failed as the sandbox was stopped: %v", sbID)
 	}
 
-	defer func() {
-		if ctx.Err() == context.Canceled || ctx.Err() == context.DeadlineExceeded {
-			log.Infof(ctx, "createCtr: context was either canceled or the deadline was exceeded: %v", ctx.Err())
-			debug.PrintStack()
```

Comment:
do we not want the PrintStack() anymore?

Reply:
nope, it doesn't give you any helpful notification. the point for it was it would point us to where the context deadline exceeded, but it just points to where we check for the context error.